rskj-core/src/main/java/org/ethereum/vm/program/Program.java

```diff
@@ -183,6 +184,13 @@ public Program(byte[] ops, ProgramInvoke programInvoke) {
         isLogEnabled = logger.isInfoEnabled();
         isGasLogEnabled =gasLogger.isInfoEnabled();
 
+        if (isLogEnabled ) {
```

Comment:
I personally prefer to not include this kind of checks. This practice was usual in the past to prevent string concatenations in the messages when a logger was not enabled, nowadays logback supports some kind of _interpolation_ using the `{}` wildcards which will not build the final log message unless the level is enabled.